packages/cashier/src/Stores/__tests__/crypto-fiat-converter-store.spec.js

```diff
@@ -0,0 +1,323 @@
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual(3);
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual('11.01');
+        expect(crypto_fiat_converter_store.converter_to_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeTrue();
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.setAccountTransferAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should reset converter details when onChangeConverterToAmount is called without passing target value', async () => {
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: '' }, 'USD', 'AED');
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.percentageSelectorSelectionStatus
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual('');
+        expect(crypto_fiat_converter_store.converter_to_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeFalse();
+    });
+
+    it('should set converter_to_amount amount', async () => {
+        WS.send.mockResolvedValue({
+            exchange_rates: {
+                base_currency: 'USD',
+                date: 1650868989,
+                rates: {
+                    AED: 3.67,
+                },
+            },
+        });
+
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: { value: 3 } }, 'USD', 'AED');
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.percentageSelectorSelectionStatus
+        ).toBeCalledTimes(1);
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.calculatePercentage
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual(3);
+    });
+
+    it('should set converter_from_error with error', async () => {
+        crypto_fiat_converter_store.setConverterToError(new Error('something went wrong'));
+
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: { value: 3 } }, 'USD', 'AED');
+
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeFalse();
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.setAccountTransferAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should set converter_from_amount with an amount', async () => {
```

Comment:
@vinu-deriv also i believe that it will be better to change the description:
'should set converter_from_amount with an amount when onChangeConverterToAmount method was called'